packages/smooth_app/lib/cards/data_cards/image_upload_card.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:io';
+import 'package:flutter/material.dart';
+
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/model/Product.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:image_cropper/image_cropper.dart';
+
+class ImageUploadCard extends StatefulWidget {
+  const ImageUploadCard({
+    Key key,
```

Comment:
We don't seem to need `Key`; please remove it unless you know for sure you'll need it.

Reply:
OK, it was just from boiler plate code I copied.